lib/Translator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+class Translator
+{
+    private const DEFAULT_LANGUAGE = 'english';
+
+    public static function loadTranslations(): void
+    {
+        global $CONFIG, $_LANG;
```

Comment:
I've got goose bumps if I see this :D Is it possible to not use `global`

Reply:
See this comment: https://github.com/plesk/whmcs-360monitoring/pull/1#discussion_r1034984336